### ISSUE TITLE
sample source / tissue description

### ISSUE THREAD

**Issue by davidsebfischer** [theislab/sfaira#179]
```
sample_source: constrained string, {"primary_tissue", "2d_culture", "3d_culture", "cancer"}
```
- [ ] Would `cancer`not better be described as `tumor`?

**Comment by le-ander** [comment]
no opinion here from my side, will rename to tumor :+1:

**Comment by le-ander** [comment]
closed in 36e759ab3fadb0edcaeb5379cfc749cbf5d17b24
